Library/SharedFunctions.swift

```diff
@@ -380,6 +380,15 @@ public func rewardsCarouselCanNavigateToReward(_ reward: Reward, in project: Pro
   let isAvailableForNewBacker = rewardIsAvailable(project: project, reward: reward) && !isBacking
   let isAvailableForExistingBackerToEdit = (isBacking && reward.hasAddOns)
 
+  if featurePostCampaignPledgeEnabled(), project.postCampaignPledgingEnabled,
+    project.isInPostCampaignPledgingPhase {
+    return [
+      project.state == .successful,
+      isAvailableForNewBacker || isAvailableForExistingBackerToEdit
```

Comment:
Can post campaign pledges be edited? I thought they couldn't, since they're charged immediately